libs/application/templates/signature-collection/signature-list-signing/src/forms/Draft.ts

```diff
@@ -74,6 +75,14 @@ export const Draft: Form = buildForm({
                     return (data as SignatureCollectionList[]).map((list) => ({
                       value: list.title,
                       label: list.title,
+
+                      disabled: list.maxReached,
+                      tag: list.maxReached
+                        ? {
+                            label: m.selectCandidateMaxReached.defaultMessage,
```

Comment:
are we able to skip `.defaultMessage` here or does it complain?

Reply:
It complains :/ Wants string